src/compose/ComposeMenu.js

```diff
@@ -183,17 +184,20 @@ class ComposeMenu extends PureComponent<Props> {
               />
             )}
             <IconImage
+              accessibilityLabel="Gallery"
               style={this.styles.composeMenuButton}
               size={24}
               onPress={this.handleImagePicker}
             />
             <IconCamera
+              accessibilityLabel="Camera"
               style={this.styles.composeMenuButton}
               size={24}
               onPress={this.handleCameraCapture}
             />
             {insertVideoCallLink !== null ? (
               <IconVideo
+                accessibilityLabel="Video Call button"
```

Comment:
I think we don't need to include the word "button" in the label. It looks like there's another way to say what kind of thing it is: the [`accessibilityRole` prop](https://reactnative.dev/docs/accessibility#accessibilityrole). If we set that prop to "button", VoiceOver on iOS automatically says "button" after reading the label.

Since the value passed to `accessibilityRole` is a special identifier, I expect it doesn't have to be translated.

Reply:
Also, I think the label could be a bit more specific about what it does. Maybe, "Insert video call link"? That way, it's clear that it's not going to, e.g., open a video call immediately. (The user might not like the idea of opening a video call immediately; they may want to prepare their environment a bit before doing so.)

The icon by itself is already ambiguous that way, and it will stay ambiguous for people who don't use a screen reader. But we might as well clarify it here, for people who do. 🙂